src/components/ContentBox.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import MovieCard from './MovieCard';
+
+export default class ContentBox extends Component {
+  render() {
+    const array = [
```

Comment:
This array should not live inside the `render()` method. Move it even outside the class beneath the imports. Now it is a just mock data, but later your component have to receive that data from props or execute API call.

Reply:
Moved.